# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/promise/Promise.h
# Commit: 0bd696ed34ea
# Full Hash: 0bd696ed34eace2844a540509a5022dc57b48508
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2019-03-27 04:44:00
# Regressor Bug: 1483036
# File Overlap Count: 2
# Description:
#   Bug 1483036 - Add MaybeResolveWithClone and MaybeRejectWithClone to Promise. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D24385
# ==============================================================================

diff -r b7663bdb3f6e -r 0bd696ed34ea dom/promise/Promise.h
--- a/dom/promise/Promise.h	Tue Mar 26 19:28:15 2019 +0000
+++ b/dom/promise/Promise.h	Tue Mar 26 19:34:29 2019 +0000
@@ -103,6 +103,9 @@
 
   void MaybeRejectWithUndefined();
 
+  void MaybeResolveWithClone(JSContext* aCx, JS::Handle<JS::Value> aValue);
+  void MaybeRejectWithClone(JSContext* aCx, JS::Handle<JS::Value> aValue);
+
   // DO NOT USE MaybeRejectBrokenly with in new code.  Promises should be
   // rejected with Error instances.
   // Note: MaybeRejectBrokenly is a template so we can use it with DOMException